### PR TITLE
Add ReST layer

### PR DESCRIPTION
**Pull request by wwguo**
The layer adds some functions to the built-in rst.el and supports Sphinx.

### CODE REVIEW

File: layers/+lang/rest/README.org [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ - [[Install][Install]]
+ - [[Key bindings][Key bindings]]
+
+* Description
+
+The layer adds ReST support to Spacemacs and adds some functions to rst-mode.
+
+** Features
+
+  - ret files support via Emacs built-in rst.el.
```
Comment (porglezomp) [inline layers/+lang/rest/README.org]: "ret" should be "rst"

**Comment by syl20bnr** [comment]
Is the code available on github or on MELPA ? I would prefer to bundle all the code as local packages if possible.

**Comment by syl20bnr** [comment]
You should create a branch for your PR instead of using the master branch.
Also we prefer PR against develop instead of master.
Tell me if you need help.

File: layers/+lang/rest/README.org (hunk 1 above)
Comment (wwguo) [inline layers/+lang/rest/README.org]: Thanks!

**Comment by wwguo** [comment]
Sorry, it is my first time of PR. How can I change the PR against develop? Should I draw back the PR and open a new one?

**Comment by syl20bnr** [comment]
I don't know how to change the target branch on GitHub, you have to open a new one I guess.

**Comment by wwguo** [comment]
I will open a new PR.

**Comment by IbnSaeed** [comment]
has this been added yet ?

**Comment by d12frosted** [comment]
@IbnSaeed this PR was closed and I haven't seen new one against develop branch.

CC @wwguo

**Comment by wwguo** [comment]
Sorry, I'm kind of busy recently. Thanks for asking! Here is the new PR, #6661.

**Pull request closed without merging**